### DISCUSSION TITLE
Brak funkcji piaskownicy

### DISCUSSION THREAD

**Discussion by Piotr2608** [AzureCommunityPL/group#433] [Pomoc]
Witajcie.
Mam mały problem, w moim koncie na portalu Azure nie mam funkcji piaskownicy.  Konto stworzone jest na te same dane co Microsoft Learn i aby przeprowadzać testy za wszystko muszę płacić. Doczytałem że rozwiązaniem jest stworzenie konta prywatnego i połączenie go z kontem na Microsoft Learn, ale ja mam zarejestrowane to właśnie jako konto prywatne i nie mogę dodać drugiego konta prywatnego.
Może ktoś ma jakiś pomysł, jak uruchomić tą piaskownicę ?

**Comment by wi5nia** [comment]
Ale Ty chcesz wejsc do sandboxa bezposrednio z subskrypcji, a nie przez MS Larn?

**Comment by smajchrz** [comment]
Możesz podać link do zadania w MS Learn gdzie masz sandbox, który Ci się nie otwiera?
